frontend/test/cypress/integration/product-list/parts_results_view.spec.ts

```diff
@@ -0,0 +1,63 @@
+describe('parts page results view', () => {
+   const user = cy;
+
+   beforeEach(() => {
+      user.visit('/Parts');
+   });
+
+   it('all products have price visible', () => {
+      user
+         .findByTestId('list-view-products')
+         .children('article')
+         .each((productListItem) => {
+            user
+               .wrap(productListItem)
+               .findByTestId('product-price')
+               .should('be.visible');
+         });
+   });
+
+   /**
+    * In ProductListItem, we set noOfLines prop to 3
+    * to truncate the product description text. This
+    * tests each product description to be less than 4
+    * lines.
+    */
+   it('product description is truncated', () => {
+      // Counting text lines https://stackoverflow.com/questions/783899
+      user.get('[data-testid="product-short-desc"]').each((descriptionDiv) => {
+         const divHeight = descriptionDiv.outerHeight() ?? 0;
+         const lineHeigth = parseInt(descriptionDiv.css('lineHeight'));
+         const noOfLines = divHeight / lineHeigth;
```

Comment:
Are we retrieving the estimated number of lines by dividing the overall div size with the line height size?

Reply:
We dropped this test case in https://github.com/iFixit/react-commerce/pull/422/commits/bc0835f7b30fcd28c34ed48fde336ce1e2936f01. I added a snapshot test for the whole component in https://github.com/iFixit/react-commerce/pull/438.